lib/attachinary/orm/file_mixin.rb

```diff
@@ -26,6 +28,15 @@ def fullpath(options={})
       format = options.delete(:format)
       Cloudinary::Utils.cloudinary_url(path(format), options.reverse_merge(:resource_type => resource_type))
     end
+
+    def preview_url(options={})
+      options.merge!(transformation: {
```

Comment:
I wonder if there is a way we can config these default opts that they aren't baked in to the gem. Non-blocking for now, just a thought.